### ISSUE TITLE
Uploading avatars to AWS S3

### ISSUE THREAD

**Issue by govind-kumarr** [govind-kumarr/simple_rag_dashboard#8]
![Screenshot (627)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/511564fa-4e60-4586-b5c4-1ec784f20a3c)

Currently, if the user is signing up using Google. His Google avatar URL is stored in the collection. But that URL is not working properly in the front end as you can see in the screenshot the URL is broken so the possible solution is to fetch the image from that URL and upload it to AWS S3 and use it from there. I have already deployed two lambdas to upload and fetch images from s3. Let me know if anyone is interested in working on this. This task is related to the backend.

**Comment by Rhythm1710** [comment]
I can work on this.

**Comment by litesh1123** [comment]
I have worked on AWS services,  i know how to upload images from web to s3 ,using lambda 

i have worked on robust architecture Amazon API
![lambda api 2](https://github.com/govind-kumarr/simple_rag_dashboard/assets/139573484/37738175-a90a-48fc-9e3e-e8278ca412f8)
 gateway connected to lambda and s3 bucket 

please assign me this issue

**Comment by litesh1123** [comment]
@govind-kumarr  thank you sir

**Comment by govind-kumarr** [comment]
@litesh1123  any update ?

**Comment by litesh1123** [comment]
@govind-kumarr  yes sir working on it ,my aws account is banned testing with new aws account
will update in 1 day

**Comment by govind-kumarr** [comment]
Hi, @litesh1123  sharing you code of my lambdas. You just need to call these lambdas at the backend. 
https://gist.github.com/govind-kumarr/e838b4745bc96b0227854bbad9b8e1cc

**Comment by govind-kumarr** [comment]
![Screenshot (631)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/a31505c0-8d7b-439b-9f80-7df71c0ad45e)

**Comment by litesh1123** [comment]
thank you sir it will be helpful

**Comment by tejaswini4996** [comment]
I would love contributing to this issue

**Comment by govind-kumarr** [comment]
@litesh1123  please update your fork and test the code. Do the modifications if needed and raise PR. 
https://github.com/govind-kumarr/simple_rag_dashboard/commit/6d9d9d607c431341bf4fc768ca984845066e4399

**Comment by govind-kumarr** [comment]
@litesh1123  I am going to close this issue as this is completed. You can raise a PR.

**Comment by litesh1123** [comment]
ok sir

**Comment by litesh1123** [comment]
i will be doing it today

**Comment by tejaswini4996** [comment]
who is working on this issue is it assigned to me?

**Comment by litesh1123** [comment]
@govind-kumarr codes fine but lambda code can be optimized and other things can be added in code file

**Comment by govind-kumarr** [comment]
@litesh1123  Sure you can add optimized lambda code in the backend I will review it and deploy it.

**Comment by litesh1123** [comment]
okay than i will be adding files with names and update it
